driver/controller.go

```diff
@@ -62,21 +62,9 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Error(codes.InvalidArgument, "CreateVolume Volume capabilities must be provided")
 	}
 
-	/*
-		if req.AccessibilityRequirements != nil {
-			for _, t := range req.AccessibilityRequirements.Requisite {
-				region, ok := t.Segments["region"]
-				if !ok {
-					continue // nothing to do
-				}
-
-				if region != d.region {
-					return nil, status.Errorf(codes.ResourceExhausted, "volume can be only created in region: %q, got: %q", d.region, region)
-
-				}
-			}
-		}
-	*/
```

Comment:
Please reinstate this. We will start using regions very soon. It's a good reminder that this is not finished.

Reply:
I reintroduced it; I wasn't aware that support for multiple regions is going to be available soon